### ISSUE TITLE
508: Live region contains a form element

### ISSUE THREAD

**Issue by vishnunavuluri** [US-EPA-CAMD/PI-1#2393]
**Issue:** If a form element is contained within the live region, the screen reader will be reading out the live region contents when the form element changes causing confusion and verbosity for screen reader users. The filter by keyword box is inside the aria-live region.  This can trigger some AT to read the entire table as each character is typed by the user into the search box.

![image.png](https://images.zenhubusercontent.com/610014e44c513c0bad9f811c/352ab3c5-40d2-452a-9c00-c336f2853e75)

**Resolution:** Move the form element out of the live region

**Comment by aprematta** [comment]
Still seeing the same behavior on this, filter by keyword is still contained in a live region

**Comment by aprematta** [comment]
ANDI is still throwing the warning. From what I can gather, I think this is fixed on desktop view, but not on the smaller responsive views. @vishnunavuluri

**Comment by aprematta** [comment]
Did testing with Tony. There is still an ANDI warning but this appears to be a false positive now. HTML does not have the issue and screen readers behave normally at different resolutions and zooming. Closing ticket
